tests/views/test_change_response_status.py

```diff
@@ -68,6 +68,9 @@ def setup_data(self):
             "EQ_LAUNCH": "INPROGRESS",
             "SUCCESSFUL_RESPONSE_UPLOAD": "COMPLETE",
             "COMPLETED_BY_PHONE": "COMPLETEDBYPHONE",
+            "RESPONDENT_ENROLED": "NOTSTARTED",
+            "ACCESS_CODE_AUTHENTICATION_ATTEMPT": "NOTSTARTED",
+            "NO_LONGER_REQUIRED": "NOLONGERREQUIRED",
```

Comment:
Does this need an assert somewhere  (NO_LONGER_REQUIRED)?

Reply:
Yeah that's fair.  Added one.